test/test_dataloader.py

```diff
@@ -1366,6 +1375,31 @@ def test_multiprocessing_contexts(self):
                 self.assertEqual(
                     reference, list(self._get_data_loader(ds_cls(counting_ds_n), multiprocessing_context=ctx, **dl_common_args)))
 
+    @skipIfNoNumpy
+    def test_multiprocessing_iterdatapipe(self):
+        # Testing to make sure that function from global scope (e.g. imported from library) can be serialized
+        # and used with multiprocess DataLoader
+        def row_processer(row):
+            return np.add(row, 1)
+
+        def filter_len(row):
+            return len(row) == 4
```

Comment:
Local function is not pickable. You need to put functions out of `test_multiprocessing_iterdatapipe`